samples/jigsaw_puzzle/jigsaw-prototype.py

```diff
@@ -7,7 +7,7 @@
 from botocore.config import Config
 from numpy import frombuffer, uint8
 
-f = open("b64/lightOn", "r")
+f = open("b64/viv", "r")
```

Comment:
Is this intentional?

Reply:
Yeah, it doesn't matter. It is a script, the filename changes based on what we are doing. It was updated to viv last since she requested a file to be uploaded to s3 through jigsaw and I used her name as filename